app/assets/javascripts/components/common/date_picker.jsx

```diff
@@ -9,10 +9,11 @@ const DatePicker = React.createClass({
 
   propTypes: {
     id: React.PropTypes.string,
-    value: React.PropTypes.string,
+    value: React.PropTypes.object, // should be a moment object
```

Comment:
I think an alternative could be:

```
React.PropTypes.shape({
  _isAMomentObject: React.PropTypes.bool
})
```
Per http://andrewhfarmer.com/validate-nested-proptypes/